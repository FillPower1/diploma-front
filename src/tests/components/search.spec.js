import React from 'react'
import { shallow } from 'enzyme'
import Search from '../../components/search'

describe('Search', () => {
	const props = {}

	describe('Cart Item render', () => {

		const nextProps = {
			...props,
		}

		const cartItem = shallow(<Search {...nextProps} />)

		it('renders properly', () => {
			expect(cartItem).toMatchSnapshot()
		})

		it('render ProductPrice', () => {
			expect(cartItem.find('.input').matchesElement(<input />)).toEqual(true)
		})
	})
})