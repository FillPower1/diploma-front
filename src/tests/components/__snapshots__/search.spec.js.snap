// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Cart Item render renders properly 1`] = `ShallowWrapper {}`;
